### PR TITLE
[owncloud] Upgrade to 10.3 version

### PR DESCRIPTION
**Pull request by pedroluislopez**
Upgrade owncloud to 10.3 version

### CODE REVIEW

File: ansible/roles/owncloud/tasks/setup_owncloud.yml [hunk 1]
```diff
@@ -158,8 +158,8 @@
     name: 'ownCloud Background Jobs'
     minute: '{{ owncloud__cron_minute }}'
     user: '{{ owncloud__app_user }}'
-    job: 'test -x /usr/bin/php && test -e {{ (owncloud__deploy_path + "/cron.php") | quote }}
-          && /usr/bin/php -f {{ (owncloud__deploy_path + "/cron.php") | quote }}'
+    job: 'test -x /usr/bin/php && test -e {{ (owncloud__deploy_path + "/occ") | quote }}
+          && /usr/bin/php {{ (owncloud__deploy_path + "/occ") | quote }} system:cron'
     cron_file: 'owncloud'
```
Comment (ypid) [inline ansible/roles/owncloud/tasks/setup_owncloud.yml:173]: This does not work for Nextcloud. Can you make this change conditional based on if Nextcloud or ownCloud is used?

Ref: https://docs.nextcloud.com/server/17/admin_manual/configuration_server/background_jobs_configuration.html

**Review by ypid — CHANGES_REQUESTED** [review]
Please do not break Nextcloud support :)

File: ansible/roles/owncloud/tasks/setup_owncloud.yml (hunk 1 above)
Comment (pedroluislopez) [reply ansible/roles/owncloud/tasks/setup_owncloud.yml:173]: Thank you. Done

**Review by ypid — APPROVED** [review]
LGTM. The ownCloud variant has not received much attention over the last years. Out of curiosity, why do you use ownCloud instead of Nextcloud? The plan was to drop ownCloud support some day.

**Comment by pedroluislopez** [comment]
Because i have a old instance of owncloud and now I don't have time to migrate to nextcloud. But i need to maintain this and i needes this upgrade.

Maybe in the future i will migrate to nextcloud.

**Review by drybjed — APPROVED** [review]
Nicely done, thanks.

**Pull request merged**